### PR TITLE
Ecto 2.0 Changes

### PR DESCRIPTION
**Pull request by veverkap**
This supplants #2

### CODE REVIEW

File: lib/new_relixir/plug/instrumentation.ex [hunk 1]
```diff
@@ -45,8 +45,15 @@ defmodule NewRelixir.Plug.Instrumentation do
...
-    infer_model(model)
+  defp infer_model(%{model: model}), do: infer_model(model)
+
+  defp infer_model(%Ecto.Changeset{data: schema}) do
+    schema 
+      |> inspect 
+      |> String.split("{")
+      |> List.first
+      |> model_name
+      |> String.replace("\"", "")
```
Comment (romul) [inline lib/new_relixir/plug/instrumentation.ex]: @veverkap It's too tricky and dirty way.
I think, `schema.__struct__ |> model_name` would be enough.
Or `schema.__struct__ |> Module.split |> List.last` as other option.

**Comment by veverkap** [comment]
@romul I switched to using `schema.__struct__ |> model_name` and it worked great!

**Comment by romul** [comment]
@veverkap If it supports both version of Ecto, then dependency should be `>= 1.1`
And I guess `git rebase origin/master -i` with squashing of commits to a small one would be cool.

**Comment by veverkap** [comment]
@romul I'm not sure that it supports both versions of Ecto

```
== Compilation error on file lib/new_relixir/plug/instrumentation.ex ==
** (CompileError) lib/new_relixir/plug/instrumentation.ex:50: unknown key :data for struct Ecto.Changeset
    (elixir) src/elixir_map.erl:185: :elixir_map."-assert_struct_keys/5-lc$^0/1-0-"/5
    (elixir) src/elixir_map.erl:62: :elixir_map.translate_struct/4
    (stdlib) lists.erl:1353: :lists.mapfoldl/3
```

**Comment by romul** [comment]
@veverkap Well, I will try it by myself.

**Comment by doomspork** [comment]
@romul do you think you will merge either this PR or #6?

**Comment by veverkap** [comment]
I'm going to close this in lieu of #6 which seems nicer.

**Comment by romul** [comment]
@doomspork I would recommend to take a look at #10 as well.
Personally I use `{:new_relixir, github: "romul/new-relixir"}` until there is no feedback to PRs.

**Comment by doomspork** [comment]
Sorry @romul!  I wrongfully assumed you were the maintainer, I now see it is @dcuddeback.

**Pull request closed without merging**